docs/core/tutorials/debugging-with-visual-studio.md

```diff
@@ -1,120 +1,124 @@
+   ![Console window showing the entered values](./media/debugging-with-visual-studio/console-window.png)
 
-1. Press any key to exit the application and end Debug mode.
+1. Press any key to exit the application and stop debugging.
 
 ---
 
-## Setting a conditional breakpoint
+## Set a conditional breakpoint
 
-Your program displays the string that the user enters. What happens if the user doesn't enter anything? You can test this with a useful debugging feature, the *conditional breakpoint*, which breaks program execution when one or more conditions are met.
+Your program displays the string that the user enters. What happens if the user doesn't enter anything? You can test this with a useful debugging feature called a *conditional breakpoint*, which breaks program execution when one or more conditions are met.
 
 To set a conditional breakpoint and test what happens when the user fails to enter a string, do the following:
 
 # [C#](#tab/csharp)
 
-1. Right-click on the red dot that represents the breakpoint. On the context menu, select **Conditions** to open the **Breakpoint Settings** dialog. Check the box for **Conditions**.
+1. Right-click on the red dot that represents the breakpoint. On the context menu, select **Conditions** to open the **Breakpoint Settings** dialog. Check the box for **Conditions** if it's not already selected.
 
    ![Editor showing breakpoint settings panel - C#](./media/debugging-with-visual-studio/breakpoint-settings.png)
 
-1. For the **Conditional Expression** replace "e.g. x == 5" with the following:
+1. For the **Conditional Expression**, replace "e.g. x == 5" with the following:
 
    ```csharp
    String.IsNullOrEmpty(name)
    ```
 
-   You're testing for a code condition, that the `String.IsNullOrEmpty(name)` method call is `true` either because *name* has not been assigned a value or because its value is an empty string (""). You can also specify a *hit count*, which interrupts program execution before a statement is executed a specified number of times, or a *filter condition*, which interrupts program execution based on such attributes as a thread identifier, process name, or thread name.
+   You're testing for a code condition, that the `String.IsNullOrEmpty(name)` method call is `true` either because `name` has not been assigned a value or because its value is an empty string (""). Instead of a conditional expression, you can also specify a *hit count*, which interrupts program execution before a statement is executed a specified number of times, or a *filter condition*, which interrupts program execution based on such attributes as a thread identifier, process name, or thread name.
 
-1. Select the **Close** button to close the dialog.
+1. Select **Close** to close the dialog.
 
-1. Run the program in Debug mode.
+1. Start the programming with debugging by pressing **F5**.
```

Comment:
That reads awkward to me:

```suggestion
1. Start the program with debugging by pressing **F5**.
```

Reply:
Oops, that was a typo. Thanks for catching!